run_schedule.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+import main
+
+def run(*args):
+    today = datetime.today()
+    day = today.weekday()
+    now = datetime.now()
+    current_hour = int(now.strftime("%H%M"))
+    
+    if day == 0 or 1 or 2 or 3 or 4 and current_hour >= 900 and current_hour <= 1500:
+        main()  #This job is run Monday-Friday 9am-3pm.
```

Comment:
ok.  Yes I forgot it was twr. then the class.  what was Stocks_and_orders_submit_stock_orders doing?  Just a method call instead of instantiating the class object?

Reply:
Apparently they both did the same thing, but it's an uncommon and longer version of the same call: https://realpython.com/instance-class-and-static-methods-demystified/